src/static/js/app/api/SurveyApi.js

```diff
@@ -0,0 +1,26 @@
+import 'fetch';
+
+export default class SurveyApi {
+  static getFirstPageForUserId(userId, callback) {
```

Comment:
Making these static so they can be called like `SurveyApi.getFirstPageForUserId` and such.